examples/gnrc_border_router/Makefile.usb-cdc-ecm.conf

```diff
@@ -0,0 +1,11 @@
+# USB Modules
+USEMODULE += usbus
+USEMODULE += auto_init_usbus
+USEMODULE += usbus_cdc_ecm
+
+ifeq (0,$(USE_DHCPV6))
```

Comment:
Why shouldn't your `start_network.sh` script also run for DHCPv6? You even added a `--use-dhcpv6` option.

Reply:
859eba1